core/src/main/java/org/mskcc/cbio/portal/model/GeneticAlterationType.java

```diff
@@ -37,7 +37,9 @@
 public enum GeneticAlterationType {
     MUTATION_EXTENDED,
     FUSION,
+    FUSION_GML,
     STRUCTURAL_VARIANT,
+    STRUCTURAL_VARIANT_GML,
```

Comment:
Is STRUCTURAL_VARIANT_GML used anywhere?

Reply:
@mandawilson not yet but there are plans to support structural variant data soon(ish) so I added it preemptively